src/Components/PostsDisplay.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useNavigate } from "react-router-dom";
+
+function PostsDisplay({
+  messages,
+  startUpdate,
+  deleteMessage,
+  addLike,
+  addDislikes,
+  isLoggedIn,
+  user
+}) {
+  const navigate = useNavigate()
+  const tdStyle = { border: "1px solid white", padding: "10px" };
+
```

Comment:
Consider using tailwind component for table. https://tailwindui.com/components/application-ui/lists/tables

This reduces the amount of custom style you need